discover/portal/articles/130-forms/10-workflow-and-forms.markdown

```diff
@@ -9,7 +9,7 @@ instance or Site level.
 
 Forms is different, and that's why its assets don't appear in the above image.
 There are so many use cases for forms, and there could be so many per site, that
-a site---or instance---scoped workflow configuration won't serve your needs 
+a site- or instance-scoped workflow configuration won't serve your needs 
```

Comment:
Should have a blank character after the first hyphen?

Reply:
@natocesarrego yeah, that's a shorthand way of saying

"a site-scoped or instance-scoped workflow configuration..."

Reply:
Nice. Thanks for the explanation! 🙂